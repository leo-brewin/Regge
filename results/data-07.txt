# defect
-4.762758197778541990E-03

# i  j   d(defect)/d(lsq(i,j))
  1  2 -2.379066343197124970E+01
  1  3 -2.381055576506754610E+01
  2  3 -2.379014257258852670E+01
  4  1  2.885059165092133510E+01
  4  2  2.881226685524994930E+01
  4  3  2.884995308486902880E+01
  5  1  3.420880044916819910E+00
  5  2  3.444100292318653230E+00
  5  3  3.430124800071366500E+00
  4  5 -2.202204154887537020E+01
  6  1  3.433180786314704700E+00
  6  2  3.434387665218968920E+00
  6  3  3.434926951450099300E+00
  5  6 -2.202553477873492830E+01
  7  1  3.446397255846553850E+00
  7  2  3.421002694977863800E+00
  7  3  3.435241513152465180E+00
  6  7 -2.200901506465982010E+01
  8  1  3.437677941758723030E+00
  8  2  3.429202822473720160E+00
  8  3  3.433943443350945010E+00
  7  8 -2.200413573557680090E+01
  9  1  2.882722313564544930E+01
  9  2  2.881008501757368740E+01
  9  3  2.882655447525365170E+01
  8  9 -2.201702700548378230E+01
  9  4 -4.189697069778266680E+01
