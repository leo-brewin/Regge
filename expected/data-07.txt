# defect
-4.762758197778536790E-03

# i  j   d(defect)/d(lsq(i,j))
  1  2 -2.379066343197124960E+01
  1  3 -2.381055576506754580E+01
  2  3 -2.379014257258852640E+01
  4  1  2.885059165092133460E+01
  4  2  2.881226685524994880E+01
  4  3  2.884995308486902830E+01
  5  1  3.420880044916819870E+00
  5  2  3.444100292318653220E+00
  5  3  3.430124800071366490E+00
  4  5 -2.202204154887537020E+01
  6  1  3.433180786314704640E+00
  6  2  3.434387665218968900E+00
  6  3  3.434926951450099290E+00
  5  6 -2.202553477873492830E+01
  7  1  3.446397255846554210E+00
  7  2  3.421002694977863810E+00
  7  3  3.435241513152465170E+00
  6  7 -2.200901506465982000E+01
  8  1  3.437677941758638790E+00
  8  2  3.429202822473720160E+00
  8  3  3.433943443350945000E+00
  7  8 -2.200413573557680090E+01
  9  1  2.882722313564544000E+01
  9  2  2.881008501757368700E+01
  9  3  2.882655447525365130E+01
  8  9 -2.201702700548378230E+01
  9  4 -4.189697069778266610E+01
